2021/Application.md

```diff
@@ -0,0 +1,71 @@
+# Application process
+Technical writers interested in participating to GSoD with ArviZ should follow these steps:
+
+1. Join the dedicated ArviZ Gitter channel for season of docs: https://gitter.im/arviz-devs/season_of_docs
+2. Send an statement of interest to `arviz.devs@gmail.com` before May 1st. More info about your statement of
+   interest at []().
+3. Fulfill the application requirements before May 10th.
+4. Be available for feedback on your proposal.
+
+## Application requirements
+Only applications from people who perform the couple tasks below will be considered.
+
+The only requirement will be ...
+
+---
+---
+
+# Selection criteria
+As ArviZ project we strive to both share statistical ideas to aid the scientific workflow and to provide the code that enables that workflow.
+We’re looking for someone who shares our interest in communicating those ideas with clear and concise communication.
+
+## Required skills
+We believe that these three skills are necessary to be successful in the GSoD project:
+* The ability to provide feedback on technical documentation in all its forms: reference docs,
+  tutorials, case studies...
+* An interest in working collaboratively with the ArviZ devs (and patience in dealing with our current documentation)
+* A willingness to identify and understand the variety of end users of ArviZ
+* Knowledge of a documentation framework
+
+## Preferred skills
+Things that will help but are not required are:
+* Programming ability (in any language)
+* Knowledge of the developer workflow in Github
```

Comment:
What is the "developer workflow in Github" is the same as knowing  the basics of git/github or something more specific?

Reply:
I think it's the basics, i.e. being able to follow the contributing guide to submit a PR without problems

Reply:
I agree but "developer workflow" sounds to me like something more advanced.